static/js/components/SharePopup.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import React from "react"
+import onClickOutside from "react-onclickoutside"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from "react-share"
+import { FacebookIcon, TwitterIcon, LinkedinIcon } from "react-share"
+
+export class SharePopupHelper extends React.Component<*> {
+  input: ?HTMLInputElement
+
+  handleClickOutside = () => {
+    const { closePopup } = this.props
+    closePopup()
+  }
+
+  selectInputText = () => {
+    if (this.input) {
+      this.input.select()
+      document.execCommand("copy")
+    }
+  }
+
+  render() {
+    const { url } = this.props
+
+    return (
+      <div className="share-popup">
+        <div className="triangle" />
+        <div className="share-title">Share a link to this post:</div>
+        <input ref={ref => (this.input = ref)} readOnly value={url} />
```

Comment:
This input hangs off the end of the popup:

![2018-07-03-164628_1920x1080_scrot](https://user-images.githubusercontent.com/28598/42243918-e6ff8cd4-7ee0-11e8-8fd6-f61ec252a0d2.png)

Reply:
Firefox seems to set a different default `box-sizing` on `<input>` tags than chrome, it should be fixed now